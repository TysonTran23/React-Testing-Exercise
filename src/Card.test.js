import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it("should render", () => {
  render(<Card />);
});

it("should match snapshot", () => {
  const { asFragment } = render(
    <Card
      caption="This is a dog"
      src="https://thumbor.forbes.com/thumbor/fit-in/900x510/https://www.forbes.com/advisor/wp-content/uploads/2023/07/top-20-small-dog-breeds.jpeg.jpg"
      currNum={4}
      totalNum={4}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
