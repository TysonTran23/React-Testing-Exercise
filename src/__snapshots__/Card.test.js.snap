// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      This is a dog
    </h4>
    <img
      alt="This is a dog"
      class="Card-image"
      src="https://thumbor.forbes.com/thumbor/fit-in/900x510/https://www.forbes.com/advisor/wp-content/uploads/2023/07/top-20-small-dog-breeds.jpeg.jpg"
    />
    <small
      class="Card-small"
    >
      Image 4 of 4.
    </small>
  </div>
</DocumentFragment>
`;
